### ISSUE TITLE
Pytorch version and the learning rate problem?

### ISSUE THREAD

**Issue by vincentgc** [wangguanzhi/LADN#9]
Hi, I have two questions with this repo.
1. which version of pytorch you used for training the LADN?
2. The learning rate for generator and discriminators is set to 0.0001, but the G_loss curve goes down and then goes up. Is this normal?

**Comment by wangguanzhi** [comment]
Hi,

1. We used PyTorch 1.1.0. The repo is not actively maintained because of limited bandwidth. But we will probably make an update next summer.
2. Yeah. The training is not that stable, and sometimes you need to train the network for more epochs until you get reasonable results.

**Comment by vincentgc** [comment]
Thanks for your answer.

By the way, the pre-trained models ("light.pth", "extreme.pth") are trained with the same hyper-parameters showed in this repo?

**Comment by wangguanzhi** [comment]
Right!

The only differences are from `local_laplacian_loss_weight` and `local_smooth_loss_weight`.

The general guideline is:
To train "light.pth", set `local_laplacian_loss_weight` and `local_smooth_loss_weight` to smaller values (<=1.0).
To train "extreme.pth", set `local_laplacian_loss_weight` and `local_smooth_loss_weight` to larger values (>=2.0).

**Comment by vincentgc** [comment]
Thanks!
